src/core/services/meta/meta.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @ngdoc service
+ * @name $$mdMeta
+ * @module material.core.meta
+ *
+ * @description
+ *
+ * A provider and a service that simplifies meta tags access
```

Comment:
Might make a note that certain `<meta>` tags (such as `charset`, `http-equiv="Content-Type"`, and I think `name="viewport"`) cannot be added dynamically because the browser will have already decided how to render (and probably rendered) the page by the time they get added.

I could not find a complete list of the ones that were editable, but this caught me the other day so it's probably not very obvious to most devs :-)